src/registry.rs

```diff
@@ -2058,2053 +1280,42 @@ impl CheckCode {
-            }
-            CheckKind::UnnecessaryListComprehensionSet => {
-                Some("Rewrite as a `set` comprehension".to_string())
-            }
-            CheckKind::UnnecessaryLiteralDict(..) => {
-                Some("Rewrite as a `dict` literal".to_string())
-            }
-            CheckKind::UnnecessaryLiteralSet(..) => Some("Rewrite as a `set` literal".to_string()),
-            CheckKind::UnnecessaryLiteralWithinTupleCall(literal) => Some({
-                if literal == "list" {
-                    "Rewrite as a `tuple` literal".to_string()
-                } else {
-                    "Remove outer `tuple` call".to_string()
-                }
-            }),
-            CheckKind::UnnecessaryLiteralWithinListCall(literal) => Some({
-                if literal == "list" {
-                    "Remove outer `list` call".to_string()
-                } else {
-                    "Rewrite as a `list` literal".to_string()
-                }
-            }),
-            CheckKind::UnnecessaryReturnNone => Some("Remove explicit `return None`".to_string()),
-            CheckKind::UnsortedImports => Some("Organize imports".to_string()),
-            CheckKind::UnusedImport(name, false, multiple) => {
-                if *multiple {
-                    Some("Remove unused import".to_string())
-                } else {
-                    Some(format!("Remove unused import: `{name}`"))
-                }
-            }
-            CheckKind::UnusedLoopControlVariable(name) => {
-                Some(format!("Rename unused `{name}` to `_{name}`"))
-            }
-            CheckKind::UnusedNOQA(..) => Some("Remove unused `noqa` directive".to_string()),
-            CheckKind::UsePEP585Annotation(name) => {
-                Some(format!("Replace `{name}` with `{}`", name.to_lowercase(),))
-            }
-            CheckKind::UsePEP604Annotation => Some("Convert to `X | Y`".to_string()),
-            CheckKind::UseFixturesWithoutParameters => {
-                Some("Remove `usefixtures` decorator or pass parameters".to_string())
-            }
-            CheckKind::UseTernaryOperator(new_code) => {
-                Some(format!("Replace if-else-block with `{new_code}`"))
-            }
-            CheckKind::UseSysExit(name) => Some(format!("Replace `{name}` with `sys.exit()`")),
-            CheckKind::UselessImportAlias => Some("Remove import alias".to_string()),
-            CheckKind::UselessMetaclassType => Some("Remove `__metaclass__ = type`".to_string()),
-            CheckKind::UselessObjectInheritance(..) => {
-                Some("Remove `object` inheritance".to_string())
-            }
-            CheckKind::UselessYieldFixture(..) => Some("Replace `yield` with `return`".to_string()),
-            CheckKind::YodaConditions(left, right) => {
-                Some(format!("Replace Yoda condition with `{left} == {right}`"))
-            }
-            _ => None,
-        }
-    }
 }
```

Comment:
I'm fine with removing `summary`. It really just existed for a few `flake8-bugbear` checks that had super long messages, and that I preserved verbatim by default.